share/picongpu/tests/CollisionsBeamRelaxation/include/picongpu/param/particle.param

```diff
@@ -134,17 +161,29 @@ namespace picongpu
             template<uint32_t numParticles>
             struct OnePositionParam
             {
-                /** Count of particles per cell at initial state
+                /** Count of macro-particles per cell at initial state
                  *
                  *  unit: none */
                 static constexpr uint32_t numParticlesPerCell = numParticles;
 
                 const InCellOffset_t inCellOffset;
             };
-            // definition of quiet particle start
-            using Center1000ppc = OnePositionImpl<OnePositionParam<1000u>>;
-            using Center100ppc = OnePositionImpl<OnePositionParam<100u>>;
+            /** Definition of start position functor that
+             * places macro-particles at the initial in-cell position defined above.
+             * Number of macro-particles per cell is defined by argument to OnePositionParam<>.
+             */
+            using Corner100ppc = OnePositionImpl<OnePositionParam<100u>>;
+            using Corner1000ppc = OnePositionImpl<OnePositionParam<1000u>>;
 
         } // namespace startPosition
+
+        /** Number of maximum macro-particles per cell during density profile evaluation.
+         *
+         * Determines the weighting of a macro particle as well as the number of
+         * macro-particles which sample the evolution of the particle distribution
+         * function in phase space.
+         */
+        constexpr uint32_t TYPICAL_PARTICLES_PER_CELL = 100u;
```

Comment:
How about? 

```suggestion
        /** Number of maximum macro-particles per cell during density profile evaluation.
         *
         * Determines the weighting of a macro particle as well as the number of
         * macro-particles which sample the evolution of the particle distribution
         * function in phase space.
         */
        constexpr uint32_t TYPICAL_PARTICLES_PER_CELL = 100u;

            /** Definition of start position functor that
             * places macro-particles at the initial in-cell position defined above.
             * Number of macro-particles per cell is defined by argument to OnePositionParam<>.
             */
            using Corner = OnePositionImpl<OnePositionParam<TYPICAL_PARTICLES_PER_CELL>>;

        } // namespace startPosition

```

Reply:
This does not work, both `Corner100ppc` and `Corner1000ppc` are used in the simulation. See `speciesInitialization.param` (https://github.com/steindev/picongpu/blob/390789d086a4c96c36f0b24d9da2af9516feeac8/share/picongpu/tests/CollisionsBeamRelaxation/include/picongpu/param/speciesInitialization.param#L50).